projects/1.18/assets/lazierae2/lazierae2/lang/zh_cn.json

```
@@ -1,66 +1,121 @@
 {
-  "itemGroup.lazierae2": "Lazier AE2",
+  "itemGroup.lazierae2": "懒人AE2",
+
   "item.lazierae2.coal_dust": "煤粉",
-  "item.lazierae2.carb_fluix_dust": "含碳福鲁伊克斯粉",
-  "item.lazierae2.resonating_gem": "谐振水晶",
-  "item.lazierae2.fluix_iron_ingot": "福鲁伊克斯镀层铁锭",
+  "item.lazierae2.carbonic_fluix_dust": "含碳福鲁伊克斯粉",
   "item.lazierae2.fluix_steel_ingot": "福鲁伊克斯钢锭",
+  "item.lazierae2.resonating_crystal": "谐振水晶",
+  "item.lazierae2.resonating_seed": "谐振种子",
+  "item.lazierae2.resonating_dust": "谐振粉",
   "item.lazierae2.logic_unit": "福鲁伊克斯逻辑单元",
-  "item.lazierae2.growth_chamber": "晶体生长仓",
+  "item.lazierae2.growth_core": "晶体生长核心",
   "item.lazierae2.universal_press": "通用压印模板",
   "item.lazierae2.parallel_printed": "并行电路板",
   "item.lazierae2.parallel_processor": "并行处理器",
-  "item.lazierae2.speculative_printed": "推测电路板",
-  "item.lazierae2.speculative_processor": "推测处理器",
-  "item.lazierae2.spec_core_1": "推测核心 x1",
-  "item.lazierae2.spec_core_2": "推测核心 x2",
-  "item.lazierae2.spec_core_4": "推测核心 x4",
-  "item.lazierae2.spec_core_8": "推测核心 x8",
-  "item.lazierae2.spec_core_16": "推测核心 x16",
-  "item.lazierae2.spec_core_32": "推测核心 x32",
-  "item.lazierae2.spec_core_64": "推测核心 x64",
-  "block.lazierae2.fluix_aggregator": "福鲁伊克斯聚合器",
-  "block.lazierae2.pulse_centrifuge": "脉冲离心机",
-  "block.lazierae2.crystal_energizer": "晶体充能器",
-  "block.lazierae2.circuit_etcher": "电路板蚀刻器",
-  "container.lazierae2.fluix_aggregator": "福鲁伊克斯聚合器",
-  "container.lazierae2.pulse_centrifuge": "脉冲离心机",
-  "container.lazierae2.crystal_energizer": "晶体充能器",
-  "container.lazierae2.circuit_etcher": "电路板蚀刻器",
-  "tooltip.lazierae2.shift_details_1": "按住",
-  "tooltip.lazierae2.shift_details_2": "Shift",
-  "tooltip.lazierae2.shift_details_3": "显示更多信息",
-  "tooltip.lazierae2.energy_buffer": "能量缓冲",
-  "tooltip.lazierae2.energy_stored": "容量:",
-  "tooltip.lazierae2.energy_rate": "消耗:",
-  "tooltip.lazierae2.upgrade_slot": "升级插槽",
-  "tooltip.lazierae2.upgrade_amount": "插槽使用:",
-  "tooltip.lazierae2.upgrade_processing_multiplier": "速度倍率:",
-  "tooltip.lazierae2.upgrade_consumption_multiplier": "能量消耗:",
-  "tooltip.lazierae2.upgrade_capacity_additional": "增加容量:",
-  "tooltip.lazierae2.item_upgrade": "已安插升级:",
-  "tooltip.lazierae2.item_energy": "能量存储:",
-  "button.lazierae2.dump_header": "释放能量",
-  "button.lazierae2.dump_info": "这将释放当前所有存储的能量！",
-  "button.lazierae2.io_header": "I/O设置",
-  "button.lazierae2.io_side": "方向:",
-  "button.lazierae2.io_mode": "模式:",
-  "button.lazierae2.extract_header": "自动抽出",
-  "button.lazierae2.extract_mode": "模式:",
-  "io_side.lazierae2.top": "上",
-  "io_side.lazierae2.left": "左",
-  "io_side.lazierae2.front": "正",
-  "io_side.lazierae2.right": "右",
-  "io_side.lazierae2.bottom": "下",
-  "io_side.lazierae2.back": "后",
-  "io_setting.lazierae2.none": "无",
+
+  "block.lazierae2.aggregator": "福鲁伊克斯聚合器",
+  "block.lazierae2.etcher": "电路板蚀刻器",
+  "block.lazierae2.grinder": "脉冲离心机",
+  "block.lazierae2.infuser": "物质注入器",
+  "block.lazierae2.requester": "ME请求者",
```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |    |

```suggestion
  "block.lazierae2.requester": "ME请求器",
```